spec/units/tab_completion/dsl_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'spec_helper'
+require 'gitsh/tab_completion/dsl'
+
+describe Gitsh::TabCompletion::DSL do
+  describe '.load' do
+    it 'builds a graph of automaton states by reading the given file' do
+      path = write_temp_completions_file([
+        'stash (apply|drop|pop|show)',
+      ].join("\n"))
```

Comment:
Err … this isn't quite what I intended. Rubocop seems to struggle with nested multiline things.